qframe_test.go

```diff
@@ -1373,6 +1373,41 @@ mon
 	})
 }
 
+func TestQFrame_ReadCSVMissingColumnName(t *testing.T) {
+	input := `,COL2
+a,1.5`
+	expectedIn := `col,COL2
+a,1.5`
+
+	out := qframe.ReadCSV(strings.NewReader(input), csv.MissingColumnNameAlias("col"))
+	expected := qframe.ReadCSV(strings.NewReader(expectedIn))
+	assertEquals(t, expected, out)
+}
+
+func TestQFrame_ReadCSVDuplicateColumnName(t *testing.T) {
+	input := `COL,COL
```

Comment:
Please update this input to something like `COL,COL,COL,KOL,KOL` to better capture the essence of the current behavior.